test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -166,3 +166,36 @@ Describe "Debug-job test" -tag "Feature" {
         $result.Command | Should be "<ScriptBlock>"
     }
 }
+
+Describe "Ampersand background test" -tag "CI" {
+    Context "Simple background job" {
```

Comment:
We should monitor how long this test takes, maybe move it to `Nightly` before merging if it's too slow from creating too many background jobs.